From b2838b050e301996e78de7ac2b0aafc4f1b51766 Mon Sep 17 00:00:00 2001
From: Wu Zhangben <wuzhangben@eswincomputing.com>
Date: Wed, 3 Sep 2025 17:25:38 +0800
Subject: [PATCH 498/540] fix:remove irq-gpios in d1_lpcpu

Changelogs:
1. irq-gpios configs in d1_lpcpu cause "unexpected IRQ trap at vector cf" in wakeup
   procedure, so remove it.

Signed-off-by: Wu Zhangben <wuzhangben@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts b/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts
index e7558437cc9eb..e51e7d7f4fdcf 100755
--- a/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts
+++ b/arch/riscv/boot/dts/eswin/eic7702-deepcomputing-fml13v03-evt.dts
@@ -515,7 +515,6 @@ &d1_lpcpu {
 	memory-region = <&d1_lpcpu_reserved>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_die1_gpio7_on_default>;
-	irq-gpios = <&d1_porta 7 IRQ_TYPE_EDGE_BOTH>;
 	control-gpio-names = "socvdd";
 	control-gpio-ports = <104 GPIO_ACTIVE_HIGH>;
 };
-- 
2.53.0

